v2/data/minimal_suite_6t_2layers/mysubsuite/mytasks.robot

```diff
@@ -1,9 +1,9 @@
 *** Test Cases ***
-TestA
+Sub Test One
     Log    This is TestA doing nothing.
 
-TestB
+Sub Test One
     Log    This is TestB doing nothing.
 
-TestC
+Sub Test One
     Log    This is TestC doing nothing.
```

Comment:
Should the names really be the same here?
Also, we should apapt the Log messages.